plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -76,12 +87,14 @@ export const TotalsCoupon = ( {
 		}
 	};
 
+	const PanelComponent = isEditor ? EditorPanel : Panel;
```

Comment:
This is a replacement for rendering a NonInteractive wrapper, instead we render a mock panel in the editor where you can still interact with a RichText placed inside it but it won't have open/close functionality.